staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -260,6 +264,9 @@ type ListOptions struct {
 	// ResourceVersion. The newest available data is preferred, but any data not older than this
 	// ResourceVersion may be served.
 	ResourceVersion string
+	// ResourceVersionMatch provides the rule for how the resource version constraint applies. Defaults to
+	// ResourceVersionMatchLegacy.
```

Comment:
ResoureVersionMatchLegacy doesn't seem to be defined anywhere

Reply:
Yes, that's right. I'll update this comment to make sense.